data/shaders/standard_phong.frag

```diff
@@ -68,10 +69,15 @@ vec3 getNormal()
     vec3 B = -normalize(cross(N, T));
     mat3 TBN = mat3(T, B, N);
 
-    return normalize(TBN * tangentNormal);
+    result = normalize(TBN * tangentNormal);
 #else
-    return normalize(normalDir);
+    result = normalize(normalDir);
 #endif
+#ifdef VSG_TWO_SIDED_LIGHTING
+    if (!gl_FrontFacing)
+        result = -result;
+#endif
+
```

Comment:
Shouldn't there be return result??

Reply:
Ack, yes. The phong shader is quite untested.